modules/asset-registry/src/lib.rs

```diff
@@ -61,21 +66,16 @@ pub mod pallet {
 		AssetIdExisted,
 	}
 
-	// SBP-M2 review: you should not include asset metadata in event while it can contain huge data
-	// amount
-	// If someone is interested there should be possibility to query it from chain state
```

Comment:
I tend to partially disagree with this statement. If we'll need it in the future for indexing, we should change it back. As Events are themselves stored (temporarily) in the state of the chain, there is little meaningful difference between emitting larger events versus emitting more smaller events.

Reply:
I don't like the idea of supporting events with bunch of data just to be able to index them. The reason for this - such approach is not intended by system design in general at the moment. Following this way would force us to fork pallets to extend them with event data we need. We would face the need of on-chain storage reads sooner or later...

Reply:
I'm fine with merging it as is. It was just a comment, I'm not expecting any actions here.